app/src/main/java/org/systers/mentorship/view/activities/FeedbackActivity.kt

```diff
@@ -88,6 +91,37 @@ class FeedbackActivity : BaseActivity(), View.OnClickListener {
             }
         }
 
+        // Getting Email id from ProfileViewModel
```

Comment:
Comments are redundant remove them please. The rest of it seems fine.

Reply:
Remove some of the redundant Comments.